src/metrics/metrics.ts

```diff
@@ -221,102 +225,131 @@ export function getAverageDistance(
   return { pieceWithHighestAverageDistance, maxAverageDistance };
 }
 
+// IN PROGRESS
```

Comment:
is this? It has a unit test?

Reply:
just something used in getMoveDistanceFunct. already in main